packages/bitcore-node/src/routes/index.ts

```diff
@@ -50,7 +50,8 @@ function getRouterFromFile(path) {
 }
 
 app.use(cors());
-app.use(LogRequest);
+app.use(LogMiddleware());
+app.use(CacheMiddleware(CacheTimes.Second));
```

Comment:
Default cache of 1 second for all endpoints.

Endpoints that should be excluded can use the no-cache middleware

CacheMiddleware(CacheTimes.None)